galaxy/wrapper/hicFindTADs.xml

```diff
@@ -76,39 +90,48 @@
+        </data>
+        <data name="boundaries" from_work_dir="galaxy_tad_prefix_boundaries.bed" format="bed"
+            label="${tool.name} on ${on_string}: Boundary positions">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="score" from_work_dir="galaxy_tad_prefix_score.bedgraph" format="bedgraph" label="${tool.name} on ${on_string}: Matrix with multi-scale TAD scores">
+        <data name="score" from_work_dir="galaxy_tad_prefix_score.bedgraph" format="bedgraph"
+            label="${tool.name} on ${on_string}: Matrix with multi-scale TAD scores">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="domains" from_work_dir="galaxy_tad_prefix_domains.bed" format="bed" label="${tool.name} on ${on_string}: TAD domains">
+        <data name="domains" from_work_dir="galaxy_tad_prefix_domains.bed" format="bed"
+            label="${tool.name} on ${on_string}: TAD domains">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="boundaries_bin" from_work_dir="galaxy_tad_prefix_boundaries_bin.bed" format="bed" label="${tool.name} on ${on_string}: Linkage information">
+        <data name="boundaries_bin" from_work_dir="galaxy_tad_prefix_boundaries.gff"
+            format="bed" label="${tool.name} on ${on_string}: Boundary information plus score">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
     </outputs>
     <tests>
-        <test>
+        <!--test>
             <param name="matrix" value="hicBuildMatrix_result1.h5" ftype="h5"/>
             <param name="mode_selector" value="TAD_score"/>
             <param name="minDepth" value="20000"/>
             <param name="maxDepth" value="60000"/>
-            <param name="step" value="10000"/>
+            <param name="step" value="100000"/>
             <output name="outFileName" file="hicFindTADs_TAD_score.tabular" ftype="tabular"/>
+            <output name="tad_score_zscore_matrix" file="tadScoreFile.tabular_zscore_matrix.h5" ftype="h5" compare="sim_size"/>
         </test>
         <test>
             <param name="tadScoreFile" value="hicFindTADs_TAD_score.tabular" ftype="tabular"/>
+            <param name="tad_score_zscore_matrix" value="tadScoreFile.tabular_zscore_matrix.h5" ftype="h5"/>
             <param name="mode_selector" value="find_TADs"/>
-            <param name="delta" value="0.001 0.01 0.03 0.05 0.1"/>
-            <param name="minBoundaryDistance" value="30000"/>
-            <param name="maxThreshold" value="0.2"/>
+            <param name="delta" value="0.002"/>
+            <param name="pvale" value="0.01"/>
+
             <output name="boundaries" file="hicFindTADs_find_boundaries.bed" ftype="bed"/>
             <output name="score" file="hicFindTADs_find_score.bedgraph" ftype="bedgraph"/>
             <output name="domains" file="hicFindTADs_find_domains.bed" ftype="bed"/>
```

Comment:
The BED files and bedgraph file are empty in my testing with the latest version.